client/src/i18n/en/stock.json

```diff
@@ -103,13 +103,13 @@
     "MONTHLY_CONSUM"  : "Average Monthly Consumption",
     "MONTHLY_CONSUMPTION" : {
       "ALGO_1"          : "Algorithm 1",
-      "ALGO_1_COMMENT"  : "The average monthly consumption is obtained by dividing the quantity consumed during the period defined by the number of days of stock for said period, and by multiplying the result obtained by 30.5",
+      "ALGO_1_COMMENT"  : "The average monthly consumption is obtained by dividing the quantity consumed during the period by the number of days having stock for the period, and by multiplying the result by 30.5.",
```

Comment:
I think "with" makes more sense than "having" here.  What do you think?

Reply:
Done